gradle/testColorOutput.gradle

```diff
@@ -1,103 +0,0 @@
-/*
- * Copyright 2008-present MongoDB, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-configure(subprojects.findAll { it.name != 'util' && it.name != 'mongo-java-driver' }) {
-    tasks.withType(Test) {
-        boolean IS_LIFECYCLE = logger.isEnabled(LogLevel.LIFECYCLE)
-        boolean IS_TRAVIS = project.hasProperty('travistest') ? project.travistest : false
-        boolean INTELLIJ_ACTIVE = System.properties.get("idea.active", false)
-        boolean CONSOLE_PLAIN = System.properties.get("org.gradle.console", "rich") == "plain"
-        boolean COLOURIZE = !INTELLIJ_ACTIVE && !CONSOLE_PLAIN
-        String ANSI_BOLD_WHITE = COLOURIZE  ?  "\u001B[0;1m"                : ""
-        String ANSI_RESET = COLOURIZE       ?  "\u001B[0m"                  : ""
-        String ANSI_BLACK = COLOURIZE       ?  "\u001B[30m"                 : ""
-        String ANSI_RED = COLOURIZE         ?  "\u001B[31m"                 : ""
-        String ANSI_GREEN = COLOURIZE       ?  "\u001B[32m"                 : ""
-        String ANSI_YELLOW = COLOURIZE      ?  "\u001B[33m"                 : ""
-        String ANSI_WHITE = COLOURIZE       ?  "\u001B[37m"                 : ""
-        String PASSED = COLOURIZE           ?  "\u2713"                     : "✓"
-        String SKIPPED = COLOURIZE          ?  "\u005F"                     : "_"
-        String FAILED = COLOURIZE           ? "\u2717" + ' [TEST FAILURE]'  : "✗ [TEST FAILURE]"
```

Comment:
Is there any way to keep this `TEST FAILURE` string?  Without it, it's hard to search for a failed test on Evergreen with the crappy search bar provided by Chrome.

Reply:
This no longer works with the new runner, so I had to remove.  The reports show you the test failure in evergreen.